src/components/Modal/__test__/findTabbableElements.spec.js

```diff
@@ -1,7 +1,7 @@
 import { JSDOM } from 'jsdom';
-import findTabbableElements from '../findTabbableElements';
+import findTabbableElements from '../../../libs/manageTab/findTabbableElements';
 
-jest.mock('./../isContentHidden', () => jest.fn(() => false));
+jest.mock('../../../libs/manageTab/isContentHidden', () => jest.fn(() => false));
```

Comment:
let's move this test to the folder its belongs (libs/manageTab)
same for all helpers moved to libs